brevbaker-dsl/build.gradle.kts

```diff
@@ -30,6 +30,7 @@ dependencies {
     // JUnit 5
     testImplementation(platform("org.junit:junit-bom:$jupiterVersion"))
     testImplementation("org.junit.jupiter:junit-jupiter")
+    testRuntimeOnly("org.junit.platform:junit-platform-launcher")
```

Comment:
Jeg aksepterer at det er dette som må til, men jeg synes at det er litt merkelig at man må inkludere en avhengighet for at IDE skal kunne kjøre tester.

Reply:
Absolutt einig. Hos meg lokalt feila det med `gradle test` frå terminalen også, utan at det gjer mykje frå eller til...